agent/probe/faccess_linux.go

```diff
@@ -215,8 +218,8 @@ func (fa *FileAccessCtrl) removeDirMarks(pid int, dirs []string) int {
 	ppath := fmt.Sprintf(procRootMountPoint, pid)
 	for _, dir := range dirs {
 		path := ppath + dir
-		if dbgError := fa.fanfd.Mark(unix.FAN_MARK_REMOVE, fa.cflag|unix.FAN_EVENT_ON_CHILD, unix.AT_FDCWD, path); dbgError != nil {
-			log.WithFields(log.Fields{"dbgError": dbgError}).Debug()
+		if err := fa.fanfd.Mark(unix.FAN_MARK_REMOVE, fa.cflag|unix.FAN_EVENT_ON_CHILD, unix.AT_FDCWD, path); err != nil && !os.IsNotExist(errors.Unwrap(err)) {
```

Comment:
why use
`errors.Unwrap(err)`
here?

Reply:
The error source is built by vendor, like 
  ` fmt.Errorf("fanotify: failed to close Fd: %w", err)`
It needs to be unwrapped before matching the real error code.